<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicEqualizerFrame</class>
 <widget class="QWidget" name="GraphicEqualizerFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#WahWahWidget {color: transparent;}</string>
  </property>
  <widget class="QToasterStompEditLCD" name="lcdDisplay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>75</y>
     <width>420</width>
     <height>115</height>
    </rect>
   </property>
   <property name="color">
    <enum>QToasterLCD::Yellow</enum>
   </property>
   <property name="currentPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="stompName">
    <string>Wah Wah</string>
   </property>
   <property name="value1Title">
    <string>80 Hz</string>
   </property>
   <property name="value2Title">
    <string>160 Hz</string>
   </property>
   <property name="value3Title">
    <string>320 Hz</string>
   </property>
   <property name="value4Title">
    <string>640 Hz</string>
   </property>
   <property name="value5Title">
    <string>Low Cut</string>
   </property>
   <property name="value6Title">
    <string>High Cut</string>
   </property>
   <property name="value7Title">
    <string>Volume</string>
   </property>
   <property name="value8Title">
    <string>Page</string>
   </property>
   <property name="value9Title">
    <string>1250 Hz</string>
   </property>
   <property name="value10Title">
    <string>2500 Hz</string>
   </property>
   <property name="value11Title">
    <string>5000 Hz</string>
   </property>
   <property name="value12Title">
    <string>10000 Hz</string>
   </property>
   <property name="value13Title">
    <string>Mix</string>
   </property>
   <property name="value14Title">
    <string>Ducking</string>
   </property>
   <property name="value15Title">
    <string/>
   </property>
   <property name="value16Title">
    <string>Page</string>
   </property>
   <property name="eqBand1Value">
    <double>-11.000000000000000</double>
   </property>
   <property name="scaleView">
    <enum>QToasterStompEditLCD::GraphicEq</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="smallDials">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>75</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page12">
    <widget class="QToasterDial" name="volumeDial">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string/>
     </property>
    </widget>
    <widget class="QToasterLookUpTableDial" name="lowCutDial">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterLookUpTableDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterLookUpTableDial::None</enum>
     </property>
     <property name="unit">
      <string>Hz</string>
     </property>
    </widget>
    <widget class="QToasterLookUpTableDial" name="highCutDial">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterLookUpTableDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterLookUpTableDial::None</enum>
     </property>
     <property name="unit">
      <string>Hz</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page22">
    <widget class="QToasterDial" name="dummy15Dial">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QToasterDial" name="duckingDial">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="minValue">
      <double>-5.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>5.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QToasterDial" name="mixDial">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Small</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>100.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>1.000000000000000</double>
     </property>
     <property name="unit">
      <string>%</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="bigDials">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>560</width>
     <height>100</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page11">
    <widget class="QToasterDial" name="band4Dial">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QToasterDial" name="band2Dial">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QToasterDial" name="band1Dial">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QToasterDial" name="band3Dial">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page21">
    <widget class="QToasterDial" name="band6Dial">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
     <property name="isActive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="band5Dial">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
    </widget>
    <widget class="QToasterDial" name="band7Dial">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
     <property name="isActive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToasterDial" name="band8Dial">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::Bi</enum>
     </property>
     <property name="minValue">
      <double>-12.000000000000000</double>
     </property>
     <property name="maxValue">
      <double>12.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
     <property name="isActive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToasterEnumDial" name="pageDial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterEnumDial::Small</enum>
   </property>
   <property name="values">
    <stringlist>
     <string>One</string>
     <string>Two</string>
    </stringlist>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToasterDial</class>
   <extends>QWidget</extends>
   <header>qtoasterdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterEnumDial</class>
   <extends>QWidget</extends>
   <header>qtoasterenumdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterLookUpTableDial</class>
   <extends>QWidget</extends>
   <header>QToasterLookUpTableDial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterStompEditLCD</class>
   <extends>QWidget</extends>
   <header>QToasterStompEditLCD.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bigDials</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smallDials</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue8(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue16(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band1Dial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band2Dial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue2(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band3Dial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue3(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band4Dial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue4(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue7(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band5Dial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue9(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue13(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>duckingDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue14(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band1Dial</sender>
   <signal>valueChanged(double)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setEqBand1Value(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band2Dial</sender>
   <signal>valueChanged(double)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setEqBand2Value(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band3Dial</sender>
   <signal>valueChanged(double)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setEqBand3Value(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band4Dial</sender>
   <signal>valueChanged(double)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setEqBand4Value(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band5Dial</sender>
   <signal>valueChanged(double)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setEqBand5Value(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band6Dial</sender>
   <signal>valueChanged(double)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setEqBand6Value(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band7Dial</sender>
   <signal>valueChanged(double)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setEqBand7Value(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band7Dial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue11(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band6Dial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue10(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band8Dial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue12(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>band8Dial</sender>
   <signal>valueChanged(double)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setEqBand8Value(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowCutDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue5(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highCutDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue6(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
